girder/molecules/server/openbabel.py

```diff
@@ -41,6 +41,15 @@ def to_inchi(str_data, in_format):
 def from_inchi(str_data, out_format):
     return convert_str(str_data, 'inchi', out_format, True)
 
+def to_smiles(str_data, in_format):
+    # This returns ["<smiles>", "chemical/x-daylight-smiles"]
+    # Keep only the first part.
+    # The smiles has returns at the end of it, and may contain
+    # a return in the middle with a common name. Get rid of
+    # all of these.
+    smiles = convert_str(str_data, in_format, 'smi')[0].strip()
```

Comment:
Does this get us canonical SMILES? I thought there was a canonical format, and would like to ensure we use that if available. See open babel [docs for details](https://openbabel.org/docs/dev/FileFormats/Canonical_SMILES_format.html) - we should prefer this and canonicalize for searches.